packages/packages/commonwell-sdk/src/client/document.ts

```diff
@@ -28,11 +29,21 @@ export async function query(
   } catch (err) {
     throw new CommonwellError(`Error querying documents`, err, additionalInfo);
   }
+  return response;
```

Comment:
nit: since it became a function, we could return on the try and not have to declare `response` and have that tslint comment - just gotta keep the await so the try/catch works. NADB, really.